NK_C_API.h

```diff
@@ -837,7 +837,8 @@ extern "C" {
 	 * @param user_pin 20 characters User PIN
 	 * @return command processing error code
 	 */
-	NK_C_API int NK_set_unencrypted_read_only(const char *user_pin);
+  [[deprecated("NK_set_unencrypted_read_only is deprecated. Use NK_set_unencrypted_read_only_admin instead")]]
```

Comment:
This does not compile in C.

Reply:
Right. This is C++ only attribute. Will check tomorrow an equivalent for C. I believe it will be `__attribute__` prefix for GCC, or some kind of general macro involving it.